documentation/src/main/asciidoc/manual-index.asciidoc

```diff
@@ -481,7 +481,7 @@ will attempt to preload everything in memory.
 |-
 |Use HQL / JPQL to index entities of a target entity type. Your query should contain only one entity
 type. Mixing this approach with the criteria restriction is not allowed. Please notice that there's
-no query validation for your input. See <<jsr-352-index-scope>> for more detail and limitations.
```

Comment:
Something is wrong with the links this generates. Missing label?

Reply:
The anchors were in the wrong place. I added a commit to fix that, too.